Makefile

```diff
@@ -83,7 +83,7 @@ examples : test/bin/writer01 test/bin/reader01
 
 unit_tests :
 	@echo -e "${YELLOW}*** Building test/bin/unit_tests ...${NORMAL}"
-	$(CXX) $(CXXFLAGS) -o test/bin/unit_tests test/src/unit_tests.cpp
+	$(CXX) $(CXXFLAGS) -std=c++11 -o test/bin/unit_tests test/src/unit_tests.cpp
```

Comment:
I'm hoping it's OK to make the tests depend on C++11, they become so much more readable. (and they better be readable, because there are not tests for the tests)

Reply:
Yeah, cool.